js/components/graph.js

```diff
@@ -784,7 +785,7 @@ function createGraph(graph) {
           if (key !== "d") d[key] = parseFloat(y[key]);
         }
         try {
-          var res = eval(customValue).toPrecision(8);
+          var res = eval(customValue).toFixed(2);
```

Comment:
Some data points have more than 2 significant numbers. I used the toPrecision(8) to correct rounding errors by JS. Do you have a specific reason to change this to toFixed(2)?

Reply:
toFixed(2) looks better. We'll use that.

Reply:
Great. I will merge back now. Thanks.